### PR TITLE
soltest on windows

### PR DESCRIPTION
**Pull request by winsvega**
Try with appveyour

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -34,8 +34,15 @@ configuration:
 # We can re-enable it when we find a way to mitigate the unreliability
 # issues.  Have automated builds be reliable is the more important thing.
 #cache: build
+#init:
+#    - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 install:
     - git submodule update --init --recursive
+    - cd deps
+    - git remote add winsvega https://github.com/winsvega/cpp-dependencies.git
```
Comment (chriseth) [inline appveyor.yml]: What does this do?

**Comment by bobsummerwill** [comment]
@chriseth Looks like Dimitry has a local edit for LevelDB which is presumably relevant for testing this change?   You should be able to just update the sub-module for this in cpp-ethereum as part of your PR, though, Dimitry, rather than needing this git funkiness.

https://github.com/winsvega/cpp-dependencies/commit/82933617b216e5bf11d50e66f8e2979e9d8440d5

**Comment by chriseth** [comment]
@bobsummerwill this is solidity, we do net need leveldb.

**Comment by bobsummerwill** [comment]
Don't shoot the messenger!   Just guessing.   Maybe it is needed by `eth.exe`?

@winsvega - what is the scoop?

**Comment by winsvega** [comment]
I've customized get dependencies script in my cpp-dependencies.git
to download and install eth binary for soltest 
here I am trying to make soltest run with eth on appveyor

**Comment by winsvega** [comment]
Ready for review 

have to specify binaries 
https://github.com/ethereum/solidity/pull/948/files#diff-b4e94328ae1f7075ce53d863b844008aR48

and this function might be removed 
https://github.com/ethereum/solidity/pull/948/files#diff-5a77d143f5c7b849e49731628dc7cfc4R36

File: appveyor.yml [hunk 2]
```diff
@@ -34,6 +34,8 @@ configuration:
 # We can re-enable it when we find a way to mitigate the unreliability
 # issues.  Have automated builds be reliable is the more important thing.
 #cache: build
+init:
+    - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```
Comment (chriseth) [inline appveyor.yml:38]: Why is that needed?

File: scripts/install_eth.cmake [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        message("Unpacking ${FILE_NAME} to ${DST_DIR}")
+        execute_process(COMMAND ${CMAKE_COMMAND} -E tar -xf ${DST_FILE}
+                        WORKING_DIRECTORY ${DST_DIR})
+    endif()
+endfunction(download_and_unpack)
+
+get_filename_component(ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}/.." ABSOLUTE)
+set(CACHE_DIR "${ROOT_DIR}/deps/cache")
+set(INSTALL_DIR "${ROOT_DIR}/deps/install")
+download_and_unpack("http://winsvega.com/files/eth.tar.gz" ${INSTALL_DIR})
```
Comment (chriseth) [inline scripts/install_eth.cmake:48]: Can you use this oner: `https://github.com/bobsummerwill/cpp-ethereum/releases/download/develop-v1.3.0.401/cpp-ethereum-develop-windows.zip` or do you require some custom changes? Please use at least `https`.

File: test/RPCSession.cpp [hunk 4]
```diff
@@ -32,6 +32,36 @@
 using namespace std;
 using namespace dev;
 
+#if defined(_WIN32)
+std::string GetLastErrorStdStr()
+{
+  DWORD error = GetLastError();
```
Comment (chriseth) [inline test/RPCSession.cpp:38]: Uses spaces for indentation.

File: test/RPCSession.cpp [hunk 5]
```diff
@@ -32,6 +32,36 @@
...
+		FORMAT_MESSAGE_FROM_SYSTEM |
+		FORMAT_MESSAGE_IGNORE_INSERTS,
+		NULL,
+		error,
+		MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+		(LPTSTR) &lpMsgBuf,
+		0, NULL );
+	if (bufLen)
+	{
+	  LPCSTR lpMsgStr = (LPCSTR)lpMsgBuf;
```
Comment (chriseth) [inline test/RPCSession.cpp:53]: Uses spaces for indentation.

File: test/RPCSession.h [hunk 6]
```diff
@@ -114,6 +114,7 @@ class RPCSession: public boost::noncopyable
 
 	std::string const& account(size_t _id) const { return m_accounts.at(_id); }
 	std::string const& accountCreateIfNotExists(size_t _id);
+	void setSleepTime(int _time) { m_sleepTime = _time; }
```
Comment (chriseth) [inline test/RPCSession.h:117]: This is a self-calibrating timeout. Why do you need to set it manually?
